### PR TITLE
added tahoma plattform, cover, sensor markdown

### PR DESCRIPTION
**Pull request by philklei**
**Description:**
added markdown for tahoma plattform, covers and sensors

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8338

### CODE REVIEW

File: source/_components/cover.tahoma.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Cover
+ha_release: 0.41
+---
+
+To use your tahoma covers in your installation, add the following to your `configuration.yaml` file:
```
Comment (Landrash) [inline source/_components/cover.tahoma.markdown]: Tahoma should most likely be capitalised.

File: source/_components/sensor.tahoma.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Cover
+ha_release: 0.41
+---
+
+To use your tahoma sensors in your installation, add the following to your `configuration.yaml` file:
```
Comment (Landrash) [inline source/_components/sensor.tahoma.markdown]: Tahoma should most likely be capitalised.

File: source/_components/tahoma.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+footer: true
+logo: somfy.png
+ha_category: Hub
+ha_release: 0.41
+---
+
+
+The `tahoma` component platform is used as an interface to the [tahomalink.com](https://www.tahomalink.com) website. It adds actually covers and the sun sensor from tahoma plattform.
+
+To use your tahoma devices in your installation, add the following to your `configuration.yaml` file:
```
Comment (Landrash) [inline source/_components/tahoma.markdown]: Tahoma should most likely be capitalised.

File: source/_components/tahoma.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+comments: false
+sharing: true
+footer: true
+logo: somfy.png
+ha_category: Hub
+ha_release: 0.41
+---
+
+
+The `tahoma` component platform is used as an interface to the [tahomalink.com](https://www.tahomalink.com) website. It adds actually covers and the sun sensor from tahoma plattform.
```
Comment (Landrash) [inline source/_components/tahoma.markdown]: Tahoma should most likely be capitalised.

**Comment by frenck** [comment]
@Landrash The parent PR is closed without merging by @balloob nine days ago.
Shouldn't this PR close as well?

**Comment by balloob** [comment]
Yep. It's difficult sometimes to keep both repo's in sync :(

**Pull request closed without merging**